### DISCUSSION TITLE
Feature request: Add support for ShopGUI+/QuantumShop

### DISCUSSION THREAD

**Discussion by zykrah** [Zrips/CMI#1546] [Enhancement]
**Description of issue or feature request:**
I would love the option of being able to use the prices of items from other shop plugins instead of having to update everything yourself in worth.yml. EG: ShopGUI+ or QuantumShop

**Cmi Version (using`/cmi version`):** 
8.5.1.4
**Server Type (Spigot/Paperspigot/etc):** 
Spigot
**Server Version (using `/ver`):** 
This server is running CraftBukkit version git-Spigot-3cb9dcb-77ca7ca (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)

**Comment by spannerman79** [comment]
Actually this would be nice to have as well @Zrips 
Using https://www.spigotmc.org/resources/shopgui-1-7-1-13.6515/

**Comment by titidelaff** [comment]
Need this feature too
